src/Benchmarks/Data/ApplicationDbContext.cs

```diff
@@ -19,17 +19,27 @@ public class ApplicationDbContext : DbContext
         public ApplicationDbContext(IOptions<AppSettings> appSettings)
         {
             _appSettings = appSettings.Value;
+            UseBatchUpdate = _appSettings.Database != "postgresql";
```

Comment:
Postgres has a problem with deadlocking when batch updates aren't sorted by key and ef doesn't emit batches as sorted.